src/main/java/org/skriptlang/skript/bukkit/recipes/elements/EffResetRecipes.java

```diff
@@ -0,0 +1,41 @@
+package org.skriptlang.skript.bukkit.recipes.elements;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.Bukkit;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Reset Recipes")
+@Description("Resets the recipes to the Minecraft-provided defaults.")
```

Comment:
In my opinion "Minecraft" should be removed from here, since DataPack recipes would still be in play.